src/Slices/UserManagement/UI/Page.test.tsx

```diff
@@ -153,21 +330,90 @@ describe("UserManagementPage", () => {
 
     expect(errorMessage).toBeVisible();
 
+    //assert that correct retry function is called
+    const retryButton = await screen.findByText("Retry");
+    await userEvent.click(retryButton);
+
+    const successView = await screen.findByLabelText("users-table");
+
+    expect(successView).toBeInTheDocument();
+
     await act(async () => {
       const results = await axe(document.body);
 
       expect(results).toHaveNoViolations();
     });
+  });
+
+  it("should display error alert view when users roles API call fails", async () => {
```

Comment:
```suggestion
  it("should display error alert view when updating roles fails", async () => {
```

Reply:
this suggestion is not adequate to what that test checks

Reply:
the original isn't better imo, it makes me think you need a call both for the user, and the roles to populate the roles column 🤔But based on the content of the test, it might be the case, and if that is, then why isn't it present in the getUser query?